app/lib/screens/history/history.dart

```diff
@@ -30,23 +27,8 @@ class TravelHistory extends StatelessWidget {
         },
       ),
       floatingActionButton: FloatingActionButton(
-        onPressed: () async {
-          try {
-            buildPdf();
-            savePdf();
-            final output = await getTemporaryDirectory();
-            String fullPath = ("${output.path}/example.pdf");
-
-            Navigator.push(
-                context,
-                MaterialPageRoute(
-                    builder: (context) => PdfPreview(
-                          path: fullPath,
-                        )));
-          } on HttpException catch (err) {
-            print('Error: $err');
-          }
-        },
+        onPressed: () => Navigator.push(
+            context, MaterialPageRoute(builder: (context) => PdfPreview())),
```

Comment:
We can give the ATR object like so when the time comes
```suggestion
            context, MaterialPageRoute(builder: (context) => PdfPreview(record = selectedRecord))),
```